docs/components/AnimatedPresence/AnimatedPresence.stories.mdx

```diff
@@ -1,5 +1,9 @@
 import { Meta } from "@storybook/addon-docs";
 
+{/* Sections of this file have been duplicated in the new docs site temporarily. */}
+{/* When updating this file, make sure to also update the corresponding notes file: */}
+{/* packages/site/src/content/[Component]/[Component]Notes.mdx */}
```

Comment:
I tried to have this as a multiline comment, so just one `{/* */}` wrapping all the text. However when I did that, the formatting changed on commit to be this (see this commit 74d6798579a1c7064eabc4ffa0caa62a0f44ca84):

```
{/_ Sections of this file have been duplicated in the new docs site temporarily.
When updating this file, make sure to also update the corresponding notes file:
packages/site/src/content/[Component]/[Component]Notes.mdx _/}
```

And that was breaking the build/not actually registering as a comment. I then also tried `<!-- -->` for multi line comment and that didn't work either. The commit's are just empty and don't even show that change.  

So I landed on this `{/* */}` for each line. Seems like single line comments are ok with this format.  This also won't live in the stories files for too long (until docs site is hard launched) so I'm not too concerned. 

If you have another idea let me know. And apologies for the messy commit messages as I was trying to figure this out.